docs/data-type-support.rst

```diff
@@ -0,0 +1,60 @@
+.. meta::
+   :description: rocPRIM API library data type support
+   :keywords: rocPRIM, ROCm, API library, API reference, data type, support
+
+.. _data-type-support:
+
+******************************************
+Data type support
+******************************************
+
+* Supported input and output types.
+
+  .. list-table:: Supported Input/Output Types
+    :header-rows: 1
+    :name: supported-input-output-types
+
+    *
+      - Input/Output Types
+      - Library Data Type
+      - AMD Supports
+    *
+      - int8
+      - int8_t
+      - 🟧
```

Comment:
What is the meaning of the orange square?

Reply:
Some API tests don't work, but probably should.

Reply:
@parbenc Please list which algorithm does not support currently the specific types.